frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -195,10 +175,64 @@ class ChartSettings extends Component {
     ).some(widget => !widget.hidden);
   }
 
+  getStyleWidget = () => {
+    const widgets = this._getWidgets();
+    const series = this._getTransformedSeries();
+    const { currentWidget } = this.state;
+    const seriesSettingsWidget =
+      currentWidget && widgets.find(w => w.id === "series_settings");
+
+    if (seriesSettingsWidget) {
+      if (currentWidget.props?.seriesKey) {
+        return {
+          ...seriesSettingsWidget,
+          props: {
+            ...seriesSettingsWidget.props,
+            initialKey: currentWidget.props.seriesKey,
+          },
+        };
+      } else if (currentWidget.props?.initialKey) {
+        const settings = this._getSettings();
+        const singleSeriesForColumn = series.find(
+          single =>
+            getColumnKey(single.data.cols[1]) ===
```

Comment:
Seems like I don't understand it fully, why do we pick the column with index `1`?

Reply:
We always check `col[1]` because that is what the column that will represent the y axis after the series has been transformed